<launch>
    <arg name="uav_id" default="0"/>
    <arg name="namespace" value="$(eval 'uav' + str(arg('uav_id')))"/>
    <node pkg="px4_application" type="uav_control" name="uav_control" output="screen" ns="$(arg namespace)">
        <rosparam command="load" file="$(find px4_application)/config/controller_config/pid_controller.yaml" />
        <rosparam command="load" file="$(find px4_application)/config/rate_config/spin_rate_config.yaml" />
        <param name="uav_id" type="int" value="$(arg uav_id)" />
	</node>
</launch>